packages/connector/src/snap.ts

```diff
@@ -73,13 +73,27 @@ export async function createVP(
 ): Promise<Result<VerifiablePresentation>> {
   await validateAndSetCeramicSession(this);
 
-  return sendSnapMethod(
+  const result = await sendSnapMethod(
```

Comment:
We can replace the `as Result<VerifiablePresentation>` with `await sendSnapMethod<Result<VerifiablePresentation>>(....)`